jerry-core/debugger/debugger.c

```diff
@@ -568,6 +568,19 @@ jerry_debugger_process_message (uint8_t *recv_buffer_p, /**< pointer the the rec
       return true;
     }
 
+    case JERRY_DEBUGGER_NO_MORE_SOURCES:
+    {
+      JERRY_DEBUGGER_CHECK_PACKET_SIZE (jerry_debugger_receive_type_t);
+
+      JERRY_CONTEXT (debugger_flags) = (uint8_t) (JERRY_CONTEXT (debugger_flags)
+                                       & ~JERRY_DEBUGGER_CLIENT_SOURCE_MODE);
+      JERRY_CONTEXT (debugger_flags) = (uint8_t) (JERRY_CONTEXT (debugger_flags)
+                                       | JERRY_DEBUGGER_CLIENT_SOURCE_CLOSED);
```

Comment:
I would prefer JERRY_DEBUGGER_CLIENT_NO_SOURCE. This source closed is strange for me.